homeassistant/components/proxmoxve/binary_sensor.py

```diff
@@ -93,4 +93,11 @@ def __init__(
     @property
     def is_on(self):
         """Return the state of the binary sensor."""
+        data = self.coordinator.data
+
+        if data is None:
+            self.available = False
```

Comment:
setting available in this way is not possible, because it is an property inherited from `BinarySensorEntity`.
I would suggest to
- in `is_on()` set state to `self._state`
- write your own property `def available()` in class `ProxmoxBinarySensor`
- let `available()` return based on `self._state`

---
Current code causes a traceback:
```
2021-03-13 13:34:35 ERROR (MainThread) [homeassistant.components.binary_sensor] Error adding entities for domain binary_sensor with platform proxmoxve
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 317, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 509, in _async_add_entity
    await entity.add_to_platform_finish()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 530, in add_to_platform_finish
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 295, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 319, in _async_write_ha_state
    sstate = self.state
  File "/usr/src/homeassistant/homeassistant/components/binary_sensor/__init__.py", line 158, in state
    return STATE_ON if self.is_on else STATE_OFF
  File "/config/custom_components/proxmoxve/binary_sensor.py", line 102, in is_on
    self.available = True
AttributeError: can't set attribute
```